content/en/docs/tasks/security/authentication/claim-to-header/index.md

```diff
@@ -0,0 +1,113 @@
+
+{{< warning >}}
+Only claims of type string, boolean, and integer are supported. Array type claims are not supported at this time.
+{{< /warning >}}
+
+{{< boilerplate experimental-feature-warning >}}
+
+## Before you begin
+
+Before you begin this task, do the following:
+
+* Familiarize yourself with [Istio end user authentication](/docs/tasks/security/authentication/authn-policy/#end-user-authentication) support.
+
+* Install Istio using [Istio installation guide](/docs/setup/install/istioctl/).
+
+* Deploy two workloads: `httpbin` and `sleep`. Deploy these in one namespace,
+for example `foo`. Both workloads run with an Envoy proxy in front of each.
+Deploy the example namespace and workloads using these commands:
+
+    {{< text bash >}}
+    $ kubectl create ns foo
+    $ kubectl label namespace foo istio-injection=enabled
+    $ kubectl apply -f @samples/httpbin/httpbin.yaml@ -n foo
+    $ kubectl apply -f @samples/sleep/sleep.yaml@ -n foo
+    {{< /text >}}
+
+* Verify that `sleep` successfully communicates with `httpbin` using this command:
+
+    {{< text bash >}}
+    $ kubectl exec "$(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name})" -c sleep -n foo -- curl http://httpbin.foo:8000/ip -sS -o /dev/null -w "%{http_code}\n"
+    200
+    {{< /text >}}
+
+{{< warning >}}
+If you don’t see the expected output, retry after a few seconds.
+Caching and propagation can cause a delay.
+{{< /warning >}}
+
+## Allow requests with valid JWT and list-typed claims
+
+1. The following command creates the `jwt-example` request authentication policy
+for the `httpbin` workload in the `foo` namespace. This policy for `httpbin` workload
+accepts a JWT issued by `testing@secure.istio.io` and copies the value of claim `foo` to a http header `X-Jwt-Claim-Foo`:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: RequestAuthentication
+    metadata:
+      name: "jwt-example"
+      namespace: foo
+    spec:
+      selector:
+        matchLabels:
+          app: httpbin
+      jwtRules:
+      - issuer: "testing@secure.istio.io"
+        jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
+        outputClaimToHeaders:
+        - header: "x-jwt-claim-foo"
+          claim: "foo"
+    EOF
+    {{< /text >}}
+
+1. Verify that a request with an invalid JWT is denied:
+
+    {{< text bash >}}
+    $ kubectl exec "$(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name})" -c sleep -n foo -- curl "http://httpbin.foo:8000/headers" -sS -o /dev/null -H "Authorization: Bearer invalidToken" -w "%{http_code}\n"
+    401
+    {{< /text >}}
+
+1. Get the JWT that sets the `iss` and `sub` keys to the same value, `testing@secure.istio.io`.
+This causes Istio to generate the attribute `requestPrincipal` with the value `testing@secure.istio.io/testing@secure.istio.io`:
```

Comment:
This is confusing. Why do we care what value Istio generates for the attribute `requestPrincipal`? All the example seems to need is the JWT with issuer `testing@secure.istio.io` and with a `foo` attribute. I would expect that is what this sentence should say this command is doing. Am I missing something?